Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -142,6 +146,23 @@ public void ConfigureServices(IServiceCollection services)
         services.AddHttpClient();
         services.AddHttpClient<GraphServiceClient>()
             .AddPolicyHandler(GetRetryPolicy());
+        services.AddAzureClients(builder =>
+        {
+            builder.AddClient<ArmClient, ArmClientOptions>(options =>
+            {
+                options.Diagnostics.IsLoggingEnabled=true;
+                options.Retry.Mode = RetryMode.Exponential;
+                options.Retry.MaxRetries = 5;
+                options.Retry.Delay = TimeSpan.FromSeconds(2);
+                var tenantId = Configuration.GetSection("AzureAd").GetValue<string>("TenantId");
+                var clientId = Configuration.GetSection("AzureAd").GetValue<string>("ClientId");
+                var clientSecret = Configuration.GetSection("AzureAd").GetValue<string>("ClientSecret");
+                var subscriptionId = Configuration.GetSection("AzureAd").GetValue<string>("SubscriptionId");
+                var creds = new ClientSecretCredential(tenantId, clientId, clientSecret);
+                var client = new ArmClient(creds, subscriptionId, options);
+                return client;
+            });
+        });
```

Comment:
Does the portal need the `ArmClient`?

Reply:
No, I had preemptively added it as I was planning to hook the budget service to the workspace settings page "budget" configuration. Will remove it for now